tests/test_hooks.py

```diff
@@ -3,17 +3,24 @@
 import warnings
 from unittest.mock import MagicMock
 
+import pytest
+
 import mmcv.runner
 
+try:
+    import torch
+    import torch.nn as nn
+    from torch.utils.data import DataLoader
+except ImportError:
+    warnings.warn('torch is not available')
+    torch = None
+
+only_if_torch_available = pytest.mark.skipif(
+    torch is None, reason='torch is not available')
```

Comment:
PyTorch will be made a compulsive requirement.

Reply:
Decorators have been removed